tkweb/apps/gallery/tests.py

```diff
@@ -0,0 +1,65 @@
+from PIL import Image as PILImage
+from django.test import TestCase
+from django.test import override_settings
+from tkweb.apps.gallery.models import Album, Image
+import tempfile
+
+
+class SimpleAlbumTest(TestCase):
+    def test_empty_album(self):
+        self.assertFalse(Album.objects.exists())
+        instance = Album.objects.create(title="Album Title", slug="album-title")
+        instance.full_clean()
```

Comment:
Burde slug ikke blive sat af clean? Dvs. `instance = Album(title="Album Title")`, `instance.full_clean()`, `instance.save()`

Reply:
Lige nu bliver den sat med js via `prepopulated_fields = { 'slug': ('title',), }` i `class AlbumAdmin(admin.ModelAdmin)`. Se [her](https://github.com/TK-IT/web/blob/master/tkweb/apps/gallery/admin.py#L42). Jeg mener at det tidligere også var i `clean()`, men at det blev fjernet fordi de to ikke lavede samme slug.